src/ui/utils/prefs.ts

```diff
@@ -27,7 +27,6 @@ pref("devtools.features.networkRequestComments", true);
 pref("devtools.features.useMultipleControllers", false);
 pref("devtools.features.multipleControllerUseSnapshots", false);
 pref("devtools.features.breakpointPanelAutocomplete", true);
-pref("devtools.features.codeHeatMaps", false);
```

Comment:
In this change, I'd probably keep the flag, but just default it to `true`. This way - we can still disable it down the line if we find that it is bad.